### ISSUE TITLE
Board editor: Sometimes air wires of same net disappear

### ISSUE THREAD

**Issue by doom-fr** [LibrePCB/LibrePCB#588]
##### VERSION / OS / ENVIRONMENT

```
LibrePCB Version: 0.1.3-unstable
Git Revision:     bd616e14d1366486cf53cad8b8c7e3e67119d960
Build Date:       2019-11-17T15:33:39
Qt Version:       5.12.3 (built against 5.12.3)
CPU Architecture: x86_64
Operating System: Ubuntu 18.04.3 LTS
Platform Plugin:  xcb
```

##### SUMMARY
Sometimes when moving components, Air Wires (the yellow lines between equipotential pads) of same net disappear.

##### STEPS TO REPRODUCE
Seams to be when two consecutive air wires of same net are aligned or quite aligned.
Not always reproductible.

![image](https://user-images.githubusercontent.com/7809481/69100246-0b893880-0a5d-11ea-9022-ebf1ffd102fa.png)

Air wires are OK. And move a little bit and ...

![image](https://user-images.githubusercontent.com/7809481/69100195-ebf21000-0a5c-11ea-8229-8673d82ac88a.png)

##### EXPECTED RESULTS
Air wires are drawn

##### ACTUAL RESULTS
Sometimes, air wires are not drawn.

**Comment by doom-fr** [comment]
Can you qualify this as a bug ?
It happens a lot.

**Comment by doom-fr** [comment]
@ubruhin Can you have a look at this ?
It is really annoying, because when you align devices, the air wire disappears.

**Comment by ubruhin** [comment]
Yeah I think I can try to fix this.

**Comment by doom-fr** [comment]
Great !

**Comment by doom-fr** [comment]
![Airwire_loss](https://user-images.githubusercontent.com/7809481/69906151-2b0a5480-13bf-11ea-8042-22aeafee5e97.gif)

**Comment by doom-fr** [comment]
![Airwire_loss](https://user-images.githubusercontent.com/7809481/69906165-6e64c300-13bf-11ea-918b-3f7679e1cd11.gif)

**Comment by doom-fr** [comment]
Just see that it also has a impact on DRC.
When the airwire is not drawn, the missing connexion error is not raised.

This is really a problem as it is so possible to forget to link some components.

Can you please fix it soon ??

**Comment by ubruhin** [comment]
Yes yes, I added it to the 0.1.4 milestone ;)

**Comment by doom-fr** [comment]
Sorry to be so impatient :-)

By this, I just want to say that now, it is not just a problem of UI but a problem with the DRC.
So, much more important.

**Comment by ubruhin** [comment]
Fixed in #687.
